src/components/Notification/Notification.stories.mdx

```diff
@@ -118,16 +118,33 @@ be rendered inside other landmark elements such as the `header` and `footer`
 landmark elements. Adding a `Notification` component inside an HTML `main` landmark
 element is acceptable.
 
+**Arial Label**<br />
```

Comment:
If you add a new line after the bold text, it should create the break. That's preferable over the `<br />` element.

Reply:
With the `<br />`, the text was not being bumped to a new line.  Two line breaks after the bold text was necessary.  But I scrapped that formatting and I am using a ### heading instead.  That said, I also added the formatting to the Resources heading.  Let's remember to do that to the same heading in other components as we move forward.